tests/test-sources/plugins/utils/arrow.nix

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
Why no `defaults` configuration ? Is it the same as `examples` ?

Reply:
Ususally, we have at least `empty` and `defaults` as test cases.
`example` is optional.
Could you please add a `defaults` test case where all of the settings are set to their default value ?

Reply:
of course these are the default :sweat: hhahha. i will look at how other modules do it.

Reply:
did you still want another use case? or is empty and defaults fine.

Reply:
Yes, maybe an `example` one with this config: https://github.com/theCat69/nvim-config/blob/247a677d74aaca49f649d552da27e8051828e9eb/lua/plugins/jumping/arrow.lua#L4

Reply:
Not sure if you wanted comments, if not that should be done.

Reply:
do you know how I can type as an unsigned int or str ?  Seems width height can be an int which makes sense. Or it cna be `auto`

Reply:
In this case it wouldn't be that because you want a regular string, not a raw string.
So, `helpers.defaultNullOpts.mkNullable (with types; either str ints.unsigned)`.